apiserver/facades/client/bundle/bundle.go

```diff
@@ -96,3 +115,24 @@ func (b *bundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 	}
 	return results, nil
 }
+
+// ExportBundle exports the current model configuration as bundle.
+func (b *BundleAPI) ExportBundle() (params.StringResult, error) {
+	model, err := b.backend.Export()
+	if err != nil {
+		return params.StringResult{}, errors.Trace(err)
+	}
+
+	if err := b.checkCanRead(model); err != nil {
```

Comment:
The auth check is done first before any business logic.
What is passed to the canRead is the model tag taken from the state object. Grab this in the NewFacade() and store as a facade attribute.

Reply:
Done